cli/ballerina-cli/src/main/resources/create_cmd_templates/default/main.bal

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
ballerina doesn't support `/**/`for comments. Nevertheless, @anupama-pathirage do we really need to have a licence header for templates. AFAIU, these templates are just to make the development easier for the user and is expected to be changed by the user. IMO, the user will not need the WSO2 license header for his package. WDYT?